src/WebAdmin.cpp

```diff
@@ -248,6 +255,23 @@ bool cWebAdmin::LoadIniFile(void)
 
 
 
+bool cWebAdmin::HasUsers(void)
```

Comment:
```suggestion
bool cWebAdmin::HasUsers()
```
`(void)` is C, `()` is C++ :)

Reply:
There are a few places where `(void)` is still used. Could be another `effort/easy` issue.

Reply:
No, the point of such easy tasks is not to offload simple mechanical work to "interns", but instead to motivate people to join. This proposition is not motivational.